src/main/kotlin/no/nav/cv/notifikasjon/StatusRepository.kt

```diff
@@ -39,13 +40,15 @@ open class StatusRepository(
     """.replace(serieMedWhitespace, " ") // Erstatter alle serier med whitespace (feks newline) med en enkelt space
 
     @Transactional(readOnly = true)
-    open fun finnSiste(aktorId: String)
-            =  entityManager.createNativeQuery(sisteQuery, StatusEntity::class.java)
-                .setParameter("aktorId", aktorId)
-                .resultStream.findFirst()
-                .map { it as StatusEntity }
-                .map { it.toStatus() }
-                .orElseGet { Status.nyBruker(aktorId) }
+    open fun finnSiste(aktorId: String) = entityManager.createNativeQuery(sisteQuery, StatusEntity::class.java)
+        .setParameter("aktorId", aktorId)
+        .resultStream
```

Comment:
Hvis det bare er en entitet som skal hentes tror jeg kanskje det er mulig å unngå `resultStream` her og heller bruke `singleResult` da vil det se sånn ut: 

```
(entityManager.createNativeQuery(sisteQuery, StatusEntity::class.java)
.setParameter("aktorId", aktorId)
.singleResult as StatusEntity?)
    ?.let { it.toStatus() }
    ?: Status.nyBruker(aktorId)
```

Men det er i og for seg det samme som skjer, så det handler bare om smak her 😊

Reply:
Ah, queryen var skjult, men nå ser jeg at den er ment å fetche flere resultater, da tror jeg det jeg skrev er irrelevant 😅

Reply:
Hehe, ja, den KAN komme med flere, så tror det vil kræsje med singleResult (men har ikke testa).